docs/docs-content/vertex/install-palette-vertex/install-on-kubernetes/install.md

```diff
@@ -424,6 +396,7 @@ reach-system:
 | `imageSwapConfig.isEKSCluster`            | Set this value to `false` if you are NOT installing VerteX on an EKS cluster.                                                                                                                       | boolean  |
 | `scar`                                    | Specify your HTTP file server values. If your HTTP file server requires credentials ensure the provided values are base64 encoded. Example of the string "admin" in base64 encoding - `YWRtaW4=`.   | object   |
 | `ingress.enabled`                         | Whether to install the Nginx ingress controller. Set this to `false` if you already have an Nginx controller deployed in the cluster.                                                               | boolean  |
+| `reach-system`                            | Set `reach-system.enabled` to `true` and configure the `reach-system.proxySettings` parameters to configure VerteX to use a network proxy in your environment                                       | object   |     | object |
```

Comment:
This table doesn't render properly, likely due to two pipelines under `reach-system` : `environment | object | | object |`

Reply:
Also, I think it would improve the layout if we aligned all items in this step with the step.